text/0000-sawtooth-sdk-swift.md

```diff
@@ -0,0 +1,214 @@
+- Feature Name: sawtooth_sdk_swift
+- Start Date: 2019-02-14
+- RFC PR:
+- Sawtooth Issue:
+
+# Summary
+[summary]: #summary
+
+Develop a Sawtooth SDK for Swift to facilitate development for client side
+Cocoa/Cocoa Touch applications. It is modeled after existing SDKs,
+implementing a Sawtooth Signing module that provides methods to generate
+private/public key pairs and sign transactions. A separate repository should
+be created to host the Swift SDK.
+
+# Motivation
+[motivation]: #motivation
+
+As the interest in Sawtooth grows, so does the demand for native mobile
+applications that can create and submit valid transactions. Additionally, the
+introduction of Hyperledger Grid has expanded the possibility for mobile use
```

Comment:
This sentence could use more context as to why Grid expanded the possibility

Reply:
Updated with a better description